lib/network_ssl/network_ssl.c

```diff
@@ -330,6 +332,12 @@ dowrite(struct network_ssl_ctx * ssl)
 	 */
 	ERR_clear_error();
 
+	/* Catch SIGPIPE. */
```

Comment:
s/Catch/Ignore/?

Reply:
I changed this, but while clarifying the commit message I noticed this in `network_write.c`:
```
        /* If we don't have MSG_NOSIGNAL, catch SIGPIPE. */
#ifdef POSIXFAIL_MSG_NOSIGNAL
        if ((oldsig = signal(SIGPIPE, SIG_IGN)) == SIG_ERR) {
                warnp("signal(SIGPIPE)");
                goto failed;
        }
#endif
```

Do you want that one updated to "ignore SIGPIPE"?